utility/gmusic_playlists_to_plex.py

```diff
@@ -40,7 +40,7 @@
 mc = Mobileclient()
 if not mc.oauth_login(device_id=Mobileclient.FROM_MAC_ADDRESS):
     mc.perform_oauth()
-
+GGMUSICLIST = mc.get_all_songs()
```

Comment:
```python
if not mc.oauth_login(device_id=Mobileclient.FROM_MAC_ADDRESS):
    mc.perform_oauth()
# ---Leave a line between---
GGMUSICLIST = mc.get_all_songs()
```

Reply:
`PLEX_MUSIC_LIBRARY = plex.library.section(MUSIC_LIBRARY_NAME)` since we use it multiple times.